in_toto/runlib.py

```diff
@@ -350,15 +350,29 @@ def execute_link(link_cmd_args, record_streams):
     - The return value of the executed command.
   """
   if record_streams:
-    return_code, stdout_str, stderr_str = \
-        securesystemslib.process.run_duplicate_streams(link_cmd_args)
+    if (quiet == False): #record_streams true, quiet false
```

Comment:
```suggestion
    if quiet is False: #record_streams true, quiet false
```
or
`if not quiet`
Alternatively, `if quiet:` with the body that you currently have under `else` should also work.

Reply:
Yeah, I can definitely clean it up a bit. I'll move it around without the parentheses and the == (old school convention habits!) Thanks!